packages/rocketchat-livechat/assets/rocket-livechat.js

```diff
@@ -8,8 +8,10 @@
 // 		h.parentNode.insertBefore(j, h);
 // 	})(window, document, 'script', 'initRocket', 'http://localhost:5000/livechat');
 // </script>
+/* globals RocketChat:true */
+/* exported RocketChat */
 
-;var RocketChat = (function(w) {
+var RocketChat = (function(w) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["RocketChat" is defined but never used](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2314070221/issues/source?bid=3199634&fileBranchId=3199636#l14)